File: typescript/src/shapes/rectangle.ts
class  Rectangle implements Shape {
    private width: number;
    private height: number;
    constructor( height: number , width: number){
        this.height = height;
        this.width = width;
    }
    computeArea():number{
        return this.width * this.height;
    }
}



export { Rectangle }

File: java/src/main/java/edu/cmu/cs/cs214/lab02/shapes/Rectangle.java
package edu.cmu.cs.cs214.lab02.shapes;

public class Rectangle implements Shape {
    private double height;
    private double width;

    private double startX;
    private double startY;
    private double startFrom;
    public Rectangle(double height, double width){
        this.height = height;
        this.width = width;
    }
    public double getArea() {
        return height * width;
    }

}
